I wish_If only/Bohdanov_hw.md

```diff
@@ -0,0 +1,41 @@
+
+If only you had came up earlier to (evaluate / estimate) all the lameness of architecture that was proposed by first year student.
+
+>Обидно, что мы не должались его прихода, иначе бы мы были в курсе на каком этапе сейчас находится проект и что нам делать с этим дальше.
+
+If only we had waited for his arrival, otherwise we would be abreast on which stage is the project right now and what we should do with it futher.
+
+> Жаль что мы так и не встретились перед отъездом и не обсудили причину твоего увольнения, у меня много мыслей на этот счет и я хотел бы поделиться ими с тобой.
+
+I wish we had met before departure and discussed a reason of your quit, I had a lot of thoughts on this topic, and I wished to share it with you
+
+>Жаль что проект завершился так и не начавшись ввиду нехватки кадров и соответствующего финансирования. Если бы все сложилось то задумка вполне могла выстрелить на рынке и стать новым хитом.
+
+If only the project has ended up before it started because of lack of personnel and corresponding funding. If everything has happened as it supposed to, plan would succeed and became a new hit. 
+
+>Он пожалел, что пришел и привел с собой друга. На его месте я бы так не поступал. Если бы он пришел вовремя, этого могло и не случиться.
+
+If only he hadn't come and brought a friend with him. I wouldn't do this on his place. If he had come right in time, it might not have happened.
+
+> Если бы не ты я не знаю что бы я тогда ответил заказчику, ты повел разговор совсем в другом направлении и нам удалось избежать серьезного разговора о горящих сроках.
+
+If only you wasn't here I don't know what I would have said to the customer yoi have led the conversation in the another direction, and we succeeded to avoid a serious conversation about pressing deadlines.
+
+> Жаль что я не могу прочесть эту книгу в оригинале, иначе бы я мог узнавать он новых тенденциях в мире разработки из первоисточника не ожидая годы пока её переведут.
+
+I wish I could red this book in the original, otherwise I could learn about new trends in a development world from the origin and not wait for decades waiting for the translation
+
+> Жаль что у нас мало времени, иначе бы я смогла заложить шикарную архитектуру и реализовать проект на твердую пятерочку. А опыт у вас в этом есть? Нет, но я читала как это можно сделать.
+
+If only we had a time for founding fancy architecture and release a project for a 5+. Did you have any experience in this? No, but I had read how to deal with it.
+
+> Я сожалею, что не могу пойти с вами в бар сегодня вечером, у меня на проекте много работы.
+
+I wish I could go with you to the bar tonight, but I have a lot of job on the project.
+
+> Жаль что я не получаю фидбэк по материалу и самим занятиям, так бы я мог хотя бы знать где я бокопорю. ;-)
+
+I wish I could receive a feedback for the material and the classes itself, so I could know where am I pissing off :)
```

Comment:
feedback is an uncountable nount
classes themselves - plural?
pissing off - means to irritate unlike fuck up